hil-test/tests/i2s.rs

```diff
@@ -144,7 +144,7 @@ mod tests {
         )
         .into_async();
 
-        let (din, dout) = ctx.dout.split();
+        let (din, dout) = unsafe { ctx.dout.split() };
```

Comment:
Aww, is there no safe replacement for this? (`Flex::new().split()`? I haven't finished reviewing the PR)

Reply:
That is half of the point of this PR.